### PR TITLE
add tapOutsideBehavior option

### PR DESCRIPTION
**Pull request by crizant**

### CODE REVIEW

**Comment by diegoveloper** [comment]
Great! just one comment. This will be a breaking change for all the users. Can we add retro-compatibility and @deprecate the use of `tapOutsideToDismiss`.

**Comment by crizant** [comment]
Done. Added `@Deprecated` annotation while maintaining compatibility.

File: lib/keyboard_actions.dart [hunk 1]
```diff
@@ -279,23 +300,26 @@ class KeyboardActionstate extends State<KeyboardActions>
...
+      return Stack(
+        children: [
+          if (widget.tapOutsideBehavior != TapOutsideBehavior.none ||
+              widget.tapOutsideToDismiss)
+            Positioned.fill(
+              child: Listener(
+                onPointerDown: (_) {
+                  _clearFocus();
+                },
+                behavior: widget.tapOutsideBehavior ==
```
Comment (diegoveloper) [inline lib/keyboard_actions.dart:312]: in the case of `widget.tapOutsideToDismiss = true`, this will use `HitTestBehavior.opaque` , right?

Comment (crizant) [reply lib/keyboard_actions.dart:312]: Sure.

**Review by diegoveloper — APPROVED** [review]
LGTM

**Comment by diegoveloper** [comment]
Thanks for the contribution @crizant

**Pull request merged**